RELEASE_NOTES.md

```diff
@@ -1,3 +1,38 @@
+## WAMR-2.4.1
+
+### Breaking Changes
+
+- lib-socket: make getaddrinfo return EAI\_ values (#4498)
```

Comment:
if you count this a breaking change, probably it's better to put #4476 in the same category for consistency.

Reply:
Added #4476 into the Breaking Changes section.